pyconph/home/templates/home/home_page.html

```diff
@@ -212,6 +212,8 @@
 
     {% include "home/_keynotes.html" %}
 
+    {% include "home/_speakers.html" %}
+
```

Comment:
Hi @EstopaceMA could i bother you adding a condition here?

```
{% if page.speakers %}
    {% include “home/_speakers.html” %}
{% endif %}
```

please add this as well for the keynotes. Otherwise looks good to me 👍

Reply:
Resolved in this commit: [24bf403](https://github.com/pythonph/pyconph-2024/pull/45/commits/24bf40302d7c63c32938b15703299d9d54171696)